### DISCUSSION TITLE
Optional "height" argument as keyword argument is invalid?

### DISCUSSION THREAD

**Discussion by chroteus** [bulletphysics/bullet3#3795] [General]
This is OK:
` visual_shape_id = pb.createVisualShape(pb.GEOM_CYLINDER, radius,[1,1,1],height,rgbaColor=[1,1,1,1], specularColor=[0.4,.4,0])`

But this fails, saying "height" is not a valid keyword argument even though it's an optional argument.
`visual_shape_id = pb.createVisualShape(pb.GEOM_CYLINDER, radius=0.5,height=0.5, rgbaColor=[1,1,1,1], specularColor=[0.4,.4,0])`

**Comment by erwincoumans** [comment]
The field is called length, docs need updating.

**Comment by chroteus** [comment]
Ah, I see. Thank you.
